interfaces/Portalicious/src/styles.scss

```diff
@@ -190,7 +190,8 @@ body {
 
   .p-element.ng-touched.ng-invalid {
     .p-inputwrapper,
-    .p-inputtext {
+    .p-inputtext,
+    &.p-inputtextarea {
```

Comment:
Hmm.. shouldn't `.p-inputtext` _also_ work with the same kind of selector?

I think the `&` is missing there as well...
See:
![image](https://github.com/user-attachments/assets/21e66eeb-a88e-458f-b191-0eedd070ae78)

Reply:
Ha, looks like for input fields we need `&.p-inputtext` and for password fields we need`.p-inputtext` and I am totally not upset with primeng for this lack of consistency